src/modules/launcher/Plugins/Microsoft.Plugin.Program/Main.cs

```diff
@@ -18,6 +19,13 @@ namespace Microsoft.Plugin.Program
 {
     public class Main : IPlugin, IPluginI18n, IContextMenu, ISavable, IReloadable, IDisposable
     {
+        private static readonly IProgramArgumentParser[] _programArgumentParsers = new IProgramArgumentParser[]
+        {
+            new DoubleDashProgramArgumentParser(),
```

Comment:
[discussion] Sort of a philosophical point, order matters here in that NoArgumentsArgumentParser is really a catch all for  when no parsers are able to handle the format.  A simple thing might be just to comment that 'NoArgumentsArgumentParser' needs to be last in the list to make it obvious to someone who wants to add a different parser.  If this was likely to be reused, we could consider implementing a `chain of responsibility` object that also allows for a default response if no 'node' in the chain can handle the given input.

Reply:
For this PR I would just add a comment.  No need to overengineer at this point, but something to consider.

Reply:
Yes the order of the _programArgumentParsers array does matter a lot! It will use the first argumentparser which is compatible, so the 'most specific' parsers should be first (and the NoArgumentsParser should always be last).

Good suggestion to keep in mind 👍